network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/api/user/self/SelfUserInfoResponseJson.kt

```diff
@@ -0,0 +1,53 @@
+package com.wire.kalium.api.tools.json.api.user.self
+
+import com.wire.kalium.api.tools.json.ValidJsonProvider
+import com.wire.kalium.network.api.UserId
+import com.wire.kalium.network.api.user.self.ManagedBy
+import com.wire.kalium.network.api.user.self.SelfUserInfoResponse
+
+object SelfUserInfoResponseJson {
+    private val jsonProvider = { serializable: SelfUserInfoResponse ->
+        """
+        |{
+        |  "id": ${serializable.id},
+        |  "qualified_id": "${serializable.qualifiedId}",
+        |  "name": "${serializable.name}",
+        |  "picture": [
+        |       {}
+        |  ]
+        |  "accent_id": ${serializable.accentId},
+        |  "assets": [
+        |       {
+        |           "key": "${serializable.assets[0].key}",
+        |           "type": "${serializable.assets[0].type}"
+        |       }
+        |  ],
+        |  "local": "${serializable.locale}",
+        |  "team": "${serializable.team}",
+        |  "managed_by": "${serializable.managedBy}"
+        |}
```

Comment:
What are these separation bar chars used for?

Reply:
it's a ```marginPrefix``` [trimMargin](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-margin.html)